tests/phpt/migration_php8/009_blank_no_warning.php

```diff
@@ -0,0 +1,20 @@
+@kphp_runtime_should_not_warn
+<?php
+
+require_once 'kphp_tester_include.php';
+
+set_migration_php8_warning(0b011);
+
+$blank = " ";
+$blank_more = "   \t";
```

Comment:
A string with leading/trailing spaces like " 42" works and is tested in other tests already, right?

Reply:
Yep, in `006_is_numeric.php`